Reflections/Week_13_Reflection.md

```diff
@@ -1,22 +1,22 @@
 1. What have you accomplished this week? (Please list each accomplishment, and explain) 
-  - I was able to conncect the Linear Actuator to the currently built system. (D4.5 shows this.)
-  - I was able to get it to work together with only a few hickups, since they were built sepertly, so new parts needed to be intergrated or moved around. (D4.5 shows this.)
+  - I was able to connect the Linear Actuator to the currently built system. (D4.5 shows this.)
+  - I was able to get it to work together with only a few hiccups, since they were built separately, so new parts needed to be integrated or moved around. (D4.5 shows this.)
   - I was able to test and show off the system working in it's entirety. (D4.5 shows this.)
 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
-  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so i moved it to under the Linear Actuator, and right beside the Lego Brain. I achived this by transfering the rotational movement of the motor with a axle and gear setup. (D4.5 shows this.)
+  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so, I moved it to under the Linear Actuator, and right beside the Lego Brain. I achieved this by transferring the rotational movement of the motor with an axle and gear setup. (D4.5 shows this.)
 3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
   - make a better platform for the weight on the Linear Actuator.
-    - Specific: The platform on the linear actutor currently can not support the weight correctly. It needs duck tape as extra support, and even then it isn't as strcutually sound. So making it where the weight can lay down without falling off when using the system will be the goal.
-    - Measurable: If the weight doesn't fall off or it doesn't move as easily loying down, then I have achived the goal.
+    - Specific: The platform on the linear actuator currently cannot support the weight correctly. It needs duct tape as extra support, and even then, it is not as structurally sound. So, making it where the weight can lay down without falling off when using the system will be the goal.
+    - Measurable: If the weight does not fall off or it does not move as easily lying down, then I have achieved the goal.
     - Achievable: This is kind achievable. The hard part is going to be making sure I don't change the setup to much so that it still works as intended. 
```

Comment:
You must give an estimate of how much time you plan to spend working on this, and when you expect results.